<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppDlg</class>
 <widget class="QFrame" name="AppDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background-color: white;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="NavView" name="lstNav">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>180</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_header_bar">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="m6000.qrc">:/Image/search.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtFilter">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777</width>
            <height>35</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>输入关键字进行搜索（姓名、电话号码、QQ 号） </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stkApp"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NavView</class>
   <extends>QListView</extends>
   <header>navview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="m6000.qrc"/>
 </resources>
 <connections/>
</ui>
